From 49b6321368e4bd3cd233d045cd09004ddd7968b2 Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Mon, 15 May 2017 14:52:00 +0800
Subject: [PATCH] keyutils: fix output format

keyutils ptest output format is incorrect, according to yocto
Development Manual
(http://www.yoctoproject.org/docs/latest/dev-manual/dev-manual.html#testing-packages-with-ptest)
5.10.6. Testing Packages With ptestThe test generates output in the format used by Automake:
<result>: <testname>
where the result can be PASS, FAIL, or SKIP, and the testname can be any
identifying string.
So we should change the test result format to match yocto ptest rules.

Upstream-Status: Inappropriate [OE ptest specific]

Signed-off-by: Li Wang <li.wang@windriver.com>
Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 tests/runtest.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tests/runtest.sh b/tests/runtest.sh
index b6eaa7c..84263fb 100644
--- a/tests/runtest.sh
+++ b/tests/runtest.sh
@@ -21,6 +21,11 @@ for i in ${TESTS}; do
     echo "### RUNNING TEST $i"
     if [[ $AUTOMATED != 0 ]] ; then
         bash ./runtest.sh
+        if [ $? != 0 ]; then
+            echo "FAIL: $i"
+        else
+            echo "PASS: $i"
+        fi
     else
         bash ./runtest.sh || exit 1
     fi
-- 
2.11.0

